api/reports/models.py

```diff
@@ -0,0 +1,42 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.conf import settings
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from rest_framework.authtoken.models import Token
+
+from datetime import date
+
+
+class Report(models.Model):
+    """
+    Report model definition
```

Comment:
This comment doe not add any value. it is in models file and named as report inherits from Model. All parts of doc string can just as easily be seen from class name